server/graphql-schema.js

```diff
@@ -1316,6 +1366,17 @@ const graphqlSchema = gql`
     findOrCreateAtVersion(input: AtVersionInput!): AtVersion!
   }
 
+  enum RequiredReportPhase {
+    IS_CANDIDATE
+    IS_RECOMMENDED
+  }
```

Comment:
Maybe this could reuse the `TestPlanVersionPhase` enum instead of introducing a new enum to reduce noise?

Reply:
Yeah I wasn't sure the logic behind the `IS_X` pattern but since the previous PR was accepted for that I decided to keep it. Can modify.

Reply:
Addressed in [1f7db13](https://github.com/w3c/aria-at-app/pull/1205/commits/1f7db13b1d0ea87271d9700310ba76e944f42e4f)